source/components/Modal/WalletProviderDafault.tsx

```diff
@@ -0,0 +1,168 @@
+      width: isHovered ? `100vw` : `2.75rem`,
+      height: isHovered ? `64px` : `2.75rem`,
+      borderTopRightRadius: isHovered ? `20px` : `999px`,
+      borderTopLeftRadius: isHovered ? `20px` : `unset`,
+      opacity: isNotVisible ? 0 : 1,
+    } as MotionStyle,
+    animation: {
+      width: isHovered ? `100vw` : `2.75rem`,
+      height: isHovered ? `64px` : `2.75rem`,
+      borderTopRightRadius: isHovered ? `20px` : `999px`,
+      borderTopLeftRadius: isHovered ? `20px` : `unset`,
+      opacity: isNotVisible ? 0 : 1,
+      transition: {
+        duration: 0.3,
+        ease: 'easeInOut',
+      },
+    } as boolean | AnimationControls | TargetAndTransition | VariantLabels,
+  };
+
+  const containerIconStyleAnimation = {
+    style: {
+      width: isHovered ? `55px` : `2.75rem`,
+      backgroundColor: isHovered ? `#476DAA` : `#4d76b8`,
+      borderTopRightRadius: isHovered ? `unset` : `999px`,
+      borderTopLeftRadius: isHovered ? `20px` : `unset`,
+    } as MotionStyle,
+    animation: {
+      backgroundColor: isHovered ? `#476DAA` : `#4d76b8`,
+      borderTopRightRadius: isHovered ? `unset` : `999px`,
+      borderTopLeftRadius: isHovered ? `20px` : `unset`,
+      transition: {
+        duration: 0.3,
+        ease: 'easeInOut',
+      },
+    } as boolean | AnimationControls | TargetAndTransition | VariantLabels,
+  };
+
+  const iconStyleAnimation = {
+    style: {
+      position: `absolute`,
+      left: isHovered ? `15px` : `5px`,
+      bottom: isHovered ? `18px` : `6px`,
+    } as MotionStyle,
+    animation: {
+      position: `absolute`,
+      left: isHovered ? `15px` : `5px`,
+      bottom: isHovered ? `18px` : `6px`,
+    } as boolean | AnimationControls | TargetAndTransition | VariantLabels,
+  };
+
+  const controller = getController();
+
+  const turnPaliAsDefault = () => {
+    setIsEnabled(!isEnabled);
+    switch (isEnabled) {
+      case true:
+        controller.wallet.removeWindowEthProperty();
+        controller.wallet.setHasEthProperty(false);
+        const dapps = Object.values(controller.dapp.getAll());
+        // disconnect from all dapps when remove window.ethereum property
+        if (dapps.length) {
+          for (const dapp of dapps) {
+            if (controller.dapp.isConnected(dapp.host))
+              controller.dapp.disconnect(dapp.host);
+          }
+        }
+        break;
+      case false:
+        controller.wallet.addWindowEthProperty();
+        controller.wallet.setHasEthProperty(true);
+        break;
+      default:
+        break;
+    }
+  };
+
+  const icon = useMemo(
+    () => (isEnabled ? 'PaliDefault' : 'PaliNotDefault'),
+    [isEnabled]
+  );
+
+  const modalText = useMemo(
+    () =>
+      isEnabled ? (
+        <span className="text-white text-sm">
+          <p>Pali was successfully set as the default wallet!</p>
+        </span>
+      ) : (
+        <span className="text-white text-sm ">
+          <p>WalletName is in use and Pali is banned. </p>
```

Comment:
WalletName shouldn't be the value here, it should show the current wallet using the window.eth
opt1 
```tsx
          <p>MetaMask is in use and Pali is banned. </p>
```
opt2 
```tsx
          <p>{walletName} is in use and Pali is banned. </p>
```